src/store/ChartState.js

```diff
@@ -39,20 +39,19 @@ class ChartState {
         this.stxx.addEventListener('layout', this.saveLayout.bind(this));
         this.stxx.addEventListener('symbolChange', this.saveLayout.bind(this));
         this.stxx.addEventListener('drawing', this.saveDrawings.bind(this));
-        this.stxx.addEventListener('move', this.scrollListener.bind(this));
+        // this.stxx.addEventListener('move', this.scrollListener.bind(this));
```

Comment:
@MahboobehMohammadi By commenting out this line, the chart doesn't show the lines between spots when scrolling the chart to left or right.

Reply:
@morteza-binary causing it will keep this issue. i think it has to be rewrite based on new changes in this PR, but i don't know how you set `maxtick` here so i just comment it here so you can check it after this is merged.
![image](https://user-images.githubusercontent.com/3080372/57610548-987da180-75a3-11e9-8525-43b1e9bf4c77.png)